node/config.go

```diff
@@ -54,6 +54,7 @@ type HTTPConfig struct {
 	HTTPSCert    []byte `json:"-"`
 
 	APIAllowedOrigins []string `json:"apiAllowedOrigins"`
```

Comment:
unrelated but I think this json tag is named weirdly because the config flag for this is actually called `http-allowed-origins`

Reply:
Yeah - we should add that in a separate PR

Reply:
PR - https://github.com/ava-labs/avalanchego/pull/1567/files